src/renderer/modals/AddAccounts/steps/StepConnectDevice.js

```diff
@@ -30,6 +27,9 @@ const StepConnectDevice = ({ currency, device, transitionTo }: StepProps) => {
       ? currency.parentCurrency.name
       : currency.name
     : undefined;
+  if (!currency) {
```

Comment:
Why do we need to remove the invariant? I feel we should never reach this step without a currency at all 🤔

Reply:
In theory, we never reach this step.
but in fact,
I found that the code in this component is triggered even if skipChooseCurrencyStep== true
because of we have this component in the following line:
https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/renderer/modals/AddAccounts/index.js#L69
I don't know how react handles this and why react reach inside the StepChooseCurrency component even if I put "steps.shift();" when skipChooseCurrencyStep== true
tell me if you have a better idea. Thanks